fdbserver/workloads/DcLag.actor.cpp

```diff
@@ -75,7 +75,7 @@ struct DcLagWorkload : TestWorkload {
 		std::vector<IPAddress> ips; // all remote process IPs
 		for (const auto& process : g_simulator->getAllProcesses()) {
 			const auto& ip = process->address.ip;
-			if (process->locality.dcId().present() && process->locality.dcId().get() == g_simulator->remoteDcId) {
+			if (process->locality.dcId().present() && process->locality.dcId().get() == g_simulator->remoteDcId.get()) {
```

Comment:
I'm not sure if remoteDcId is guaranteed to be present, but we could probably write it like this and not worry about it:

```suggestion
			if (process->locality.dcId().present() && process->locality.dcId() == g_simulator->remoteDcId) {
```